grakn-test/src/test/java/ai/grakn/test/graql/reasoner/AtomicQueryTest.java

```diff
@@ -102,6 +102,23 @@ public void testCopyConstructor(){
     }
 
     @Test
+    public void testCopyConstructor2(){
```

Comment:
Maybe this could be two tests? `whenModifyingAQuery_TheCopyDoesNotChange` and `whenCopyingAQuery_TheyHaveTheSameRoleVarTypeMaps`